assets/css/_route_filter.scss

```diff
@@ -1,34 +1,60 @@
-.m-route-filter {
-  padding-bottom: 1.5rem;
-}
-
 .m-route-filter__text {
   position: relative;
+  display: flex;
+  flex-direction: column;
 }
 
 .m-route-filter__input {
-  width: 100%;
-  font-size: 0.75rem;
-  border: 1px solid #9fa2ac;
-  border-radius: 4px;
-  margin: 1px 0;
+  @include font-small;
+  line-height: 1;
+  border: 1px solid #b3b5bd;
+  border-radius: 6px;
+  padding: 0.625rem 1rem;
+  color: #3c3f4c;
```

Comment:
**suggestion (non-blocking):** This value is used in several places. It might be worthwhile turning it into a variable.